draft-ietf-webtrans-http2.md

```diff
@@ -215,6 +215,28 @@ control. This can reduce latency for data sent by a client at the start of a
 WebTransport session. For example, a client might choose to send datagrams or
 flow control updates before receiving any response from the server.
 
+## Subprotocol Negotiation
+
+WebTransport over HTTP/2 offers a subprotocol negotiation mechanism, similar to
+TLS Application-Layer Protocol Negotiation Extension (ALPN) {{?RFC7301}}; the
+intent is to simplify porting pre-existing protocols that rely on this
+functionality.
+
+The user agent MAY include a `WebTransport-Subprotocols-Available` header field
+in the CONNECT request, enumerating the possible subprotocols. If the server
+receives such a header, it MAY include a `WebTransport-Subprotocol` field in a
+successful (2xx) response. If it does, the server MUST include a single
+subprotocol from the client's list in that field. Servers MAY reject the
+request if the client did not include a suitable subprotocol.
+
+Both `WebTransport-Subprotocols-Available` and `WebTransport-Subprotocol` are
```

Comment:
Should we instead point to the h3 draft for the definition of the fields? It feels weird to redefine them here